cloudinit/distros/netbsd.py

```diff
@@ -90,8 +90,7 @@ def set_passwd(self, user, passwd, hashed=False):
         if hashed:
             hashed_pw = passwd
         else:
-            method = crypt.METHOD_BLOWFISH  # pylint: disable=E1101
-            hashed_pw = crypt.crypt(passwd, crypt.mksalt(method))
+            hashed_pw = bcrypt.hash("password")
```

Comment:
Is there a reason the password is now a static string and no salt is used? Would `bcrypt.hashpw(passwd, bcrypt.gensalt())` work?

Reply:
@goneri Could you please update this? As @tobhe pointed out, we need to pass in the variable `passwd`, rather that using the static password `"password"`, as you do in this PR.